UCSLibraries/src/main/java/it/cnr/iit/xacmlutilities/Category.java

```diff
@@ -40,7 +40,7 @@
     ENVIRONMENT( "urn:oasis:names:tc:xacml:3.0:attribute-category:environment" );
 
     // the string representing the category in xacmlformat
-    private final String xacmlString;
+    private final String data;
```

Comment:
what do you imply by data? Is it the payload?

Reply:
yep

Reply:
OK can you rename it as payloadData may be?